easybuild/easyconfigs/m/MbedTLS/MbedTLS-2.26.0-GCCcore-10.2.0.eb

```diff
@@ -17,8 +17,8 @@ checksums = ['37949e823c7e1f6695fc56858578df355da0770c284b1c1304cfc8b396d539cd']
 
 builddependencies = [
     ('binutils', '2.35'),
-    # Perl, Python only needed for test suite
-    ('Python', '3.8.6'),
+    # Perl, Python only needed for test suite (python2 required)
+    ('Python', '2.7.18'),
```

Comment:
Python 3 can be used, but we need to add `PYTHON=python` to the `buildopts`. This is because the `Makefile` has:

```Makefile
# python2 for POSIX since FreeBSD has only python2 as default.
PYTHON ?= python2
```

I've tested that on a system with:
```
python     python2.7  python3    python3.8
```
i.e. no `python2` executable. I do not have a system with no `python3`, so I cannot test if switching to using Python 2 as a build dep will cause a problem elsewhere.

Reply:
Or, possibly:
```
PYTHON=$EBROOTPYTHON/bin/python
```

Reply:
`PYTHON=python` works, fixed in 0b159b765f, thanks for the suggestion!